components/src/core/components/List/types.ts

```diff
@@ -0,0 +1,126 @@
+interface Visible {
+  visible?: boolean;
+}
+
+interface ListEmitEvent {
+  type: string;
+  identifier: string;
+}
+
+interface Tooltip {
+  tooltip?: string;
+  flow?: string;
+}
+
+interface Style {
+  [key: string]: string | undefined;
+}
+
+interface Icon {
+  iconName?: string;
+  "icon-size"?: string;
+  iconSize?: string;
+  iconStyle?: string;
+}
+
+interface ButtonIcon extends Icon {
+  "icon-name"?: string;
+}
+
+interface ListButtonProps extends Tooltip, ButtonIcon {
+  label?: string;
+  displayType?: string;
+  size?: string;
+  "display-type"?: string;
```

Comment:
Why do we have both "display-type" and "displayType" options here?

Reply:
removed the duplicate and proceed with camel case. made the relevant reference changes on Vue App